### PR TITLE
Newsweek homepage responsive clone

### PR DESCRIPTION
**Pull request by AlanSoto31**
It is a clone from Newsweek home page but including fully responsive design. One very important requisite for this project was the use of bootstrap as more as possible, which was a great opportunity to learn how this usefull tool works. It was built from mobile-size to xl-size. The most responsive design happens inside the following ranges 320px 768px 1024px 1044px.

### CODE REVIEW

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle:

## :wave: :wave: :wave: Hello Team!! Great work on the project!!

It looks really close to the original! Excelent job Team! :clap: :clap:

:100: Great Pull Request and Commit Messages
:100: Descriptive enough README
:100: Clone Very similar to the original

## Required Changes
### [Mandatory] :red_circle: 

- [x] menu doesn't resize properly on bigger screens (1440)
image

- [x] navbar is not sticky
image

- [x] Search is not vertical centered
image

- [x] More stories is not displaying properly
image

- [x] Sign up form Is not displaying properly.
image

- [x] Columns width causing trouble with elements. See images above.

- [x] Sports section not displaying properly on tablet (768)
image

- [x] White margin on mobile version (320)
image

- [x] Sign up section not displaying properly on mobile
image

- [x] Sports section not displaying properly on mobile
image

- [ ] Use semantic HTML for ```header``` and ```main```

### [OPTIONAL] :large_blue_circle: 

- [ ] Assign the ```style.css``` to it's own folder. Create a directory structure similar to this.
```
/   css/
        style.css
    img/
        some_images.jpg
    js/
        some_javascript.js
    index.html
```

- [ ] Move the ```style.css``` below the Boostrap CDN. This way makes it easier to overide Bootstrap css and use a custom stylesheet.

## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

File: index.html [hunk 1]
```diff
@@ -0,0 +1,3577 @@
...
+    />
+    <script
+      src="https://kit.fontawesome.com/d6bf1861cd.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Document</title>
+  </head>
+
+  <body>
+    <div class="body">
```
Comment (jubaan) [inline index.html]: - [ ] Wrap ```navbar``` around ```header``` semantic html tag. Body wrapper seems unnecesary if the code uses semantic html.

**Review by jubaan — CHANGES_REQUESTED** [review]

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,481 @@
...
+.whole-container {
+  max-width: 1400px;
+}
+
+.body {
+  background-color: #f4f5f7;
+}
+
+a {
+  text-decoration: none !important;
```
Comment (umairarshadbutt) [inline styles.css]: remove `!important`

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,481 @@
...
+}
+
+.itm-text-box p {
+  background: #f72210;
+}
+
+.itm-title:hover {
+  color: black;
+  text-decoration: underline !important;
+  text-decoration-color: #f72210 !important;
```
Comment (umairarshadbutt) [inline styles.css:348]: remove `!important`

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,481 @@
...
+  .ep-img {
+    height: 146px;
+  }
+
+  .nl-img {
+    height: 330px;
+  }
+
+  .nl-btn {
+    font-size: 14px !important;
```
Comment (umairarshadbutt) [inline styles.css]: remove `!important`

**Comment by umairarshadbutt** [comment]
##
## Pull Request under review by Umair Arshad (TSE)

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi @AlanSoto31 AND @NataJenkins 

Good job with the changes! :tada: We are almost there.
We, however, do have some few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [x] Remove `!important `from css file. ([Article for support](https://speckyboy.com/good-bad-css-practices/)) ( [screenshot](https://user-images.githubusercontent.com/29689273/93404399-8997b500-f8a3-11ea-8f88-fb12b8f6ca80.png) from Pathwright)
### Navbar
- [x] Remove hamburger from large screens

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://alansoto31.github.io/Newsweek-Responsive-Clone/)|
|:---:|:---:|
|source-page|clone|

### Footer
- [x] Remove the padding left

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://alansoto31.github.io/Newsweek-Responsive-Clone/)|
|:---:|:---:|
|source-page|clone|

##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you stick at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by jubaan — APPROVED** [review]
# Status: Approved :tada: :tada: :tada:

## Congrats team! you've worked on the required changes!!

You may now merge to master.

## :rocket: See you at the top!

### [COMMENTS]

- [ ] I've seen that you minimize the use of ```!important``` in the project CSS. For next projects try the project to use none. While it works and does what is expected, in the long run and bigger websites ```!important``` creates more issues than the one's it solves.

**Pull request merged**